### ISSUE TITLE
Notifications not working due to snapshot or clip not being available

### ISSUE THREAD

**Issue by IkerGarcia** [0x2142/frigate-notify#137]
**Describe the bug**
Not getting notifications after updating Frigate to 0.14. However, I can see the events in Frigate (Proxmox) and with Home Assistant.

**Expected behavior**
Getting notifications with clip and snapshot as before updating Frigate to 0.14.

**To Reproduce**
-
**Versions:**
 - Frigate NVR: [0.14]
 - Frigate-Notify: [v0.3.4] (tried both stable and dev)

**Frigate-Notify Config:**
```yaml
frigate:
  server: ****
  ignoressl:
  public_url:
  headers:
  startup_check:
    attempts:
    interval:

  webapi:
    enabled: False
    interval: 

  mqtt: 
    enabled: True
    server: ****
    port: ****
    clientid: notify
    username: ****
    password:  ****
    topic_prefix: frigate 

  cameras:
    exclude:
      - test_cam_01

alerts:  
  general:
    title:
    timeformat:
    nosnap:

  zones:
    unzoned: allow
    allow:
     - zone_0
    block:
     - test_zone_02

  labels:
    allow:
     - person
     - dog
    block:
     - bird

  discord:
    enabled: false
    webhook: 
    template:

  gotify:
    enabled: false
    server:
    token: 
    ignoressl:
    template:

  smtp:
    enabled: false
    server: 
    port: 
    tls: 
    user:
    password:
    recipient:
    template:

  telegram:
    enabled: false
    chatid:
    token:
    template:

  pushover:
    enabled: false
    token:
    userkey:
    devices:
    priority:
    retry:
    expire:
    ttl:
    template:

  ntfy:
    enabled: True
    server: ****
    topic: frigate
    ignoressl:
    headers:
    template:

  webhook:
    enabled: false
    server: 
    ignoressl:
    headers:
    template:

monitor:
  enabled: false
  url: 
  interval: 
  ignoressl:
```

**Frigate-Notify Logs:**
```yaml
2024/09/26 23:30:17 +0200 INF Frigate Notify - v0.3.4
2024/09/26 23:30:17 +0200 INF Starting...
2024/09/26 23:30:17 +0200 INF Config loaded.
2024/09/26 23:30:17 +0200 INF Successfully connected to http://192.168.1.111:5000
2024/09/26 23:30:17 +0200 INF Config file validated!
2024/09/26 23:30:17 +0200 INF App running. Press Ctrl-C to quit.
2024/09/26 23:30:17 +0200 INF Connected to MQTT.
2024/09/26 23:30:17 +0200 INF Subscribed to MQTT topic: frigate/events
2024/09/26 23:32:40 +0200 INF New event received event_id=1727386359.746662-7hdh9y
2024/09/26 23:32:40 +0200 INF Event Detected camera=garaje event_id=1727386359.746662-7hdh9y label=person zones=
2024/09/26 23:32:40 +0200 INF Event dropped - No snapshot or clip available event_id=1727386359.746662-7hdh9y
2024/09/26 23:32:40 +0200 INF Event update received event_id=1727386359.746662-7hdh9y
2024/09/26 23:32:40 +0200 INF Event Detected camera=garaje event_id=1727386359.746662-7hdh9y label=person zones=
2024/09/26 23:32:40 +0200 INF Event dropped - Already notified on this zone camera=garaje event_id=1727386359.746662-7hdh9y label=person zones=
2024/09/26 23:32:46 +0200 INF New event received event_id=1727386365.908437-0zftc0
2024/09/26 23:32:46 +0200 INF Event Detected camera=garaje event_id=1727386365.908437-0zftc0 label=person zones=
2024/09/26 23:32:46 +0200 INF Event dropped - No snapshot or clip available event_id=1727386365.908437-0zftc0
2024/09/26 23:32:46 +0200 INF Event update received event_id=1727386365.908437-0zftc0
2024/09/26 23:32:46 +0200 INF Event Detected camera=garaje event_id=1727386365.908437-0zftc0 label=person zones=
2024/09/26 23:32:46 +0200 INF Event dropped - Already notified on this zone camera=garaje event_id=1727386365.908437-0zftc0 label=person zones=
```

**Additional context**
Only one camera, motion mask, not zones.

I check the clips file and the snapshot does exist (see screenshots).

**Screenshots**
![Screenshot 2024-09-26 23 47 37](https://github.com/user-attachments/assets/e6f760bf-801c-48b7-b501-624734e90e65)

**Comment by Mareckil** [comment]
Possibly my problem is the same I also do have frigate 0.14.

**Comment by IkerGarcia** [comment]
Hi,

Thank you for the quick reply and fix. I remotely updated to v0.3.5. However I need to be there to activate the motion, I'll came back here soon.

**Comment by IkerGarcia** [comment]
Working! I'll just post the logs so you can see actual behaviour. 

``
2024/09/27 17:05:18 +0200 INF Frigate Notify - v0.3.5-dev
2024/09/27 17:05:18 +0200 INF Starting...
2024/09/27 17:05:18 +0200 INF Config loaded.
2024/09/27 17:05:18 +0200 INF Successfully connected to http://192.168.1.111:5000
2024/09/27 17:05:18 +0200 INF Config file validated!
2024/09/27 17:05:18 +0200 INF App running. Press Ctrl-C to quit.
2024/09/27 17:05:18 +0200 INF Connected to MQTT.
2024/09/27 17:05:18 +0200 INF Subscribed to MQTT topic: frigate/events
2024/09/27 19:11:41 +0200 INF New event received event_id=1727457100.352984-g0hw1c
2024/09/27 19:11:41 +0200 INF Event Detected camera=garaje event_id=1727457100.352984-g0hw1c label=person zones=
2024/09/27 19:11:41 +0200 INF Event dropped - No snapshot or clip available event_id=1727457100.352984-g0hw1c
2024/09/27 19:11:41 +0200 INF Event update received event_id=1727457100.352984-g0hw1c
2024/09/27 19:11:41 +0200 INF Event Detected camera=garaje event_id=1727457100.352984-g0hw1c label=person zones=
2024/09/27 19:11:41 +0200 INF Alert sent event_id=1727457100.352984-g0hw1c provider=Ntfy
2024/09/27 19:11:47 +0200 INF Event update received event_id=1727457100.352984-g0hw1c
2024/09/27 19:11:47 +0200 INF Event Detected camera=garaje event_id=1727457100.352984-g0hw1c label=person zones=
2024/09/27 19:11:47 +0200 INF Event dropped - Already notified on this zone camera=garaje event_id=1727457100.352984-g0hw1c label=person zones=
2024/09/27 19:12:12 +0200 INF New event received event_id=1727457132.007978-ylv80h
2024/09/27 19:12:12 +0200 INF Event Detected camera=garaje event_id=1727457132.007978-ylv80h label=person zones=
2024/09/27 19:12:12 +0200 INF Event dropped - No snapshot or clip available event_id=1727457132.007978-ylv80h
2024/09/27 19:12:13 +0200 INF Event update received event_id=1727457132.007978-ylv80h
2024/09/27 19:12:13 +0200 INF Event Detected camera=garaje event_id=1727457132.007978-ylv80h label=person zones=
2024/09/27 19:12:13 +0200 INF Alert sent event_id=1727457132.007978-ylv80h provider=Ntfy
2024/09/27 19:12:19 +0200 INF Event update received event_id=1727457132.007978-ylv80h
2024/09/27 19:12:19 +0200 INF Event Detected camera=garaje event_id=1727457132.007978-ylv80h label=person zones=
2024/09/27 19:12:19 +0200 INF Event dropped - Already notified on this zone camera=garaje event_id=1727457132.007978-ylv80h label=person zones=
2024/09/27 19:13:12 +0200 INF New event received event_id=1727457190.178922-t4nh2z
2024/09/27 19:13:12 +0200 INF Event Detected camera=garaje event_id=1727457190.178922-t4nh2z label=person zones=
2024/09/27 19:13:12 +0200 INF Event dropped - No snapshot or clip available event_id=1727457190.178922-t4nh2z
2024/09/27 19:13:13 +0200 INF Event update received event_id=1727457190.178922-t4nh2z
2024/09/27 19:13:13 +0200 INF Event Detected camera=garaje event_id=1727457190.178922-t4nh2z label=person zones=
2024/09/27 19:13:13 +0200 INF Alert sent event_id=1727457190.178922-t4nh2z provider=Ntfy
2024/09/27 19:13:20 +0200 INF New event received event_id=1727457198.657951-7sy7rs
2024/09/27 19:13:20 +0200 INF Event Detected camera=garaje event_id=1727457198.657951-7sy7rs label=person zones=
2024/09/27 19:13:20 +0200 INF Event dropped - No snapshot or clip available event_id=1727457198.657951-7sy7rs
2024/09/27 19:13:20 +0200 INF Event update received event_id=1727457198.657951-7sy7rs
2024/09/27 19:13:20 +0200 INF Event Detected camera=garaje event_id=1727457198.657951-7sy7rs label=person zones=
2024/09/27 19:13:20 +0200 INF Alert sent event_id=1727457198.657951-7sy7rs provider=Ntfy
2024/09/27 19:13:24 +0200 INF New event received event_id=1727457202.52572-jpb32b
2024/09/27 19:13:24 +0200 INF Event Detected camera=garaje event_id=1727457202.52572-jpb32b label=person zones=
2024/09/27 19:13:24 +0200 INF Event dropped - No snapshot or clip available event_id=1727457202.52572-jpb32b
2024/09/27 19:13:25 +0200 INF Event update received event_id=1727457202.52572-jpb32b
2024/09/27 19:13:25 +0200 INF Event Detected camera=garaje event_id=1727457202.52572-jpb32b label=person zones=
2024/09/27 19:13:25 +0200 INF Alert sent event_id=1727457202.52572-jpb32b provider=Ntfy
2024/09/27 19:13:27 +0200 INF Event update received event_id=1727457198.657951-7sy7rs
2024/09/27 19:13:27 +0200 INF Event Detected camera=garaje event_id=1727457198.657951-7sy7rs label=person zones=
2024/09/27 19:13:27 +0200 INF Event dropped - Already notified on this zone camera=garaje event_id=1727457198.657951-7sy7rs label=person zones=
2024/09/27 19:13:53 +0200 INF New event received event_id=1727457232.161868-38jqm8
2024/09/27 19:13:53 +0200 INF Event Detected camera=garaje event_id=1727457232.161868-38jqm8 label=person zones=
2024/09/27 19:13:53 +0200 INF Event dropped - No snapshot or clip available event_id=1727457232.161868-38jqm8
2024/09/27 19:13:54 +0200 INF Event update received event_id=1727457232.161868-38jqm8
2024/09/27 19:13:54 +0200 INF Event Detected camera=garaje event_id=1727457232.161868-38jqm8 label=person zones=
2024/09/27 19:13:54 +0200 INF Alert sent event_id=1727457232.161868-38jqm8 provider=Ntfy
2024/09/27 19:14:29 +0200 INF New event received event_id=1727457268.826359-3fubdi
2024/09/27 19:14:29 +0200 INF Event Detected camera=garaje event_id=1727457268.826359-3fubdi label=person zones=
2024/09/27 19:14:29 +0200 INF Event dropped - No snapshot or clip available event_id=1727457268.826359-3fubdi
2024/09/27 19:14:29 +0200 INF Event update received event_id=1727457268.826359-3fubdi
2024/09/27 19:14:29 +0200 INF Event Detected camera=garaje event_id=1727457268.826359-3fubdi label=person zones=
2024/09/27 19:14:29 +0200 INF Alert sent event_id=1727457268.826359-3fubdi provider=Ntfy
2024/09/27 19:14:40 +0200 INF New event received event_id=1727457279.986025-tndzis
2024/09/27 19:14:40 +0200 INF Event Detected camera=garaje event_id=1727457279.986025-tndzis label=person zones=
2024/09/27 19:14:40 +0200 INF Event dropped - No snapshot or clip available event_id=1727457279.986025-tndzis
2024/09/27 19:14:41 +0200 INF Event update received event_id=1727457279.986025-tndzis
2024/09/27 19:14:41 +0200 INF Event Detected camera=garaje event_id=1727457279.986025-tndzis label=person zones=
2024/09/27 19:14:41 +0200 INF Alert sent event_id=1727457279.986025-tndzis provider=Ntfy``

**Comment by smartelnovo** [comment]
just to confirm, I had this bug too and the dev fixed it, good job!

**Comment by 0x2142** [comment]
Thank you! I really appreciate the feedback 😊

**Comment by 0x2142** [comment]
Closing, as this is now released in [v0.3.5](https://github.com/0x2142/frigate-notify/releases/tag/v0.3.5)
